cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/BaseHiveExploreService.java

```diff
@@ -203,7 +202,6 @@ protected BaseHiveExploreService(TransactionSystemClient txClient, DatasetFramew
                                    AuthenticationContext authenticationContext) {
     this.cConf = cConf;
     this.hConf = hConf;
-    this.hiveConf = createHiveConf();
```

Comment:
this commit was causing failures?

Reply:
The Hive on Spark failure? No, it is mainly caused by the hack through the `ResourcesClassLoader`, which get removed in this PR.

Reply:
I mean - why'd you revert this caching change, then?

Reply:
It causes some unit-test failure, apparently the hiveConf can't be shared between sessions. I haven't dig deep into why, but instead reverting to a new instance